### PR TITLE
Fix(pool_exporter): remove pool_exporter container spec

### PR DESCRIPTION
**Pull request by utkarshmani1997**
Signed-off-by: Utkarsh Mani Tripathi <utkarsh.tripathi@mayadata.io>

**What this PR does / why we need it**:
This removes pool-exporter from maya-exporter's runtask. Some testing needs to be done that's why we are droping it from 0.8.x.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `breaking-changes` tag
- [ ] PR messages has breaking changes related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] PR messages has upgrade related information
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: pkg/install/v1alpha1/cstor_pool.go [hunk 1]
```diff
@@ -310,46 +295,6 @@ spec:
...
-            volumeMounts:
-            - mountPath: /dev
-              name: device
-            - mountPath: /tmp
-              name: tmp
-            - mountPath: {{ .Config.SparseDir.value }}
-              name: sparse
-            - mountPath: /run/udev
-              name: udev
-          tolerations:
```
Comment (AmitKumarDas) [inline pkg/install/v1alpha1/cstor_pool.go]: why are the tolerations being removed

File: pkg/install/v1alpha1/cstor_pool.go [hunk 2]
```diff
@@ -285,6 +267,9 @@ spec:
                 {{ $rKey }}: {{ $rLimit }}
               {{- end }}
               {{- end }}
+            ports:
```
Comment (AmitKumarDas) [inline pkg/install/v1alpha1/cstor_pool.go]: Is this relevant to pool exporter image?
Why is this added?

File: pkg/install/v1alpha1/cstor_pool.go [hunk 3]
```diff
@@ -176,7 +172,7 @@ spec:
     {{- $auxResourceRequestsVal := fromYaml .Config.AuxResourceRequests.value -}}
     {{- $setAuxResourceLimits := .Config.AuxResourceLimits.value | default "none" -}}
     {{- $auxResourceLimitsVal := fromYaml .Config.AuxResourceLimits.value -}}
-    apiVersion: apps/v1beta1
+    apiVersion: extensions/v1beta1
```
Comment (AmitKumarDas) [inline pkg/install/v1alpha1/cstor_pool.go]: was this change done as part of pool exporter & hence reverted back

File: pkg/install/v1alpha1/cstor_pool.go [hunk 4]
```diff
@@ -186,15 +182,6 @@ spec:
         app: cstor-pool
         openebs.io/version: {{ .CAST.version }}
         openebs.io/cas-template-name: {{ .CAST.castName }}
-      annotations:
```
Comment (AmitKumarDas) [inline pkg/install/v1alpha1/cstor_pool.go]: why are these removed?
cc @ashishranjan738 @vishnuitta

**Comment by utkarshmani1997** [comment]
Closing this, new PR #1035 have been raised which has latest changes also.
